src/components/simulator/SimulatorSummaryTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactComponent as Arrow } from 'assets/icons/arrow-round.svg';
+import { SimulatorResultSearch } from 'libs/routing';
+import { prettifyNumber } from 'utils/helpers';
+import { Token } from 'libs/tokens';
+
+interface Props {
+  summaryData: Pick<
+    SimulatorResultSearch,
+    'sellMin' | 'sellMax' | 'buyMin' | 'buyMax' | 'buyBudget' | 'sellBudget'
+  >;
+  baseToken: Token;
+  quoteToken: Token;
+}
+
+export const SimulatorSummaryTable = ({
+  baseToken,
+  quoteToken,
+  summaryData,
+}: Props) => {
+  const sellMin = prettifyNumber(summaryData.sellMin, { abbreviate: true });
+  const sellMax = prettifyNumber(summaryData.sellMax, { abbreviate: true });
+  const buyMin = prettifyNumber(summaryData.buyMin, { abbreviate: true });
+  const buyMax = prettifyNumber(summaryData.buyMax, { abbreviate: true });
+  const baseBudget = prettifyNumber(summaryData.sellBudget, {
+    abbreviate: true,
+  });
+  const quoteBudget = prettifyNumber(summaryData.buyBudget, {
+    abbreviate: true,
+  });
+  const baseBudgetFormatted = prettifyNumber(baseBudget, { abbreviate: true });
+  const quoteBudgetFormatted = prettifyNumber(quoteBudget, {
+    abbreviate: true,
+  });
+  const baseSymbol = baseToken.symbol;
+  const quoteSymbol = quoteToken.symbol;
+
+  return (
+    <article className="grid grid-cols-[auto,auto] grid-rows-4 items-center justify-evenly gap-2 md:grid-cols-[auto,auto,auto,auto] md:grid-rows-2">
```

Comment:
I think it might be better to use a table here.

Reply:
Addressed in cb7b9dcba3eee92d9dd14c0fcef31caedc1dea67